pkg/microservice/aslan/core/code/service/repo.go

```diff
@@ -77,7 +77,7 @@ func ListRepoInfos(infos []*GitRepoInfo, param string, log *zap.SugaredLogger) (
 			if info.Source == CodeHostCodeHub {
 				projectName = info.RepoUUID
 			}
-			info.Branches, err = CodeHostListBranches(info.CodehostID, projectName, strings.Replace(info.Owner, "%2F", "/", -1), log)
+			info.Branches, err = CodeHostListBranches(info.CodehostID, projectName, strings.Replace(info.Owner, "%2F", "/", -1), 0, 0, "", log)
```

Comment:
Why don't you provide paging capabilities in `ListRepoInfos`?

Reply:
infos dont improve this time , the frontend may change something